packages/wrangler/src/__tests__/cloudchamber/modify.test.ts

```diff
@@ -55,45 +92,30 @@ describe("cloudchamber modify", () => {
 	it("should modify deployment (detects no interactivity)", async () => {
 		setIsTTY(false);
 		setWranglerConfig({});
-		msw.use(
-			http.patch(
-				"*/deployments/1234/v2",
-				async ({ request }) => {
-					expect(await request.text()).toMatchInlineSnapshot(
-						`"{\\"image\\":\\"hello:modify\\",\\"environment_variables\\":[{\\"name\\":\\"HELLO\\",\\"value\\":\\"WORLD\\"},{\\"name\\":\\"YOU\\",\\"value\\":\\"CONQUERED\\"}],\\"vcpu\\":3,\\"memory\\":\\"40MB\\"}"`
-					);
-					return HttpResponse.json(MOCK_DEPLOYMENTS_COMPLEX[0]);
-				},
-				{ once: true }
-			)
-		);
+		mockDeployment();
 		await runWrangler(
-			"cloudchamber modify 1234 --image hello:modify --var HELLO:WORLD --var YOU:CONQUERED --vcpu 3 --memory 40MB"
+			"cloudchamber modify 1234 --image hello:modify --location sfo06 --var HELLO:WORLD --var YOU:CONQUERED --vcpu 3 --memory 40MB"
 		);
 		expect(std.err).toMatchInlineSnapshot(`""`);
 		// so testing the actual UI will be harder than expected
 		// TODO: think better on how to test UI actions
-		expect(std.out).toMatchInlineSnapshot(`
-		"{
-		    \\"id\\": \\"1\\",
-		    \\"type\\": \\"default\\",
-		    \\"created_at\\": \\"123\\",
-		    \\"account_id\\": \\"123\\",
-		    \\"vcpu\\": 4,
-		    \\"memory\\": \\"400MB\\",
-		    \\"version\\": 1,
-		    \\"image\\": \\"hello\\",
-		    \\"location\\": {
-		        \\"name\\": \\"sfo06\\",
-		        \\"enabled\\": true
-		    },
-		    \\"network\\": {
-		        \\"ipv4\\": \\"1.1.1.1\\"
-		    },
-		    \\"placements_ref\\": \\"http://ref\\",
-		    \\"node_group\\": \\"metal\\"
-		}"
-	`);
+		expect(std.out).toMatchInlineSnapshot(EXPECTED_RESULT);
+	});
+
+	it("should modify deployment with wrangler args (detects no interactivity)", async () => {
```

Comment:
one thing I'm wondering about...I noticed that all existing tests are testing the non-interactive use case. What about if `isTTY = true`? Is that a code path that should be tested?

Reply:
not blocking as we can address that as a fast follow to this PR. Just wanted to call it out

Reply:
Yeah, I think we do it later just to keep scope small. Figuring out testing the interactive mode will be tricky and I'm okay with the small lack of coverage there for now (was manually tested pretty well).